ghost/core/core/server/services/members/exporter/query.js

```diff
@@ -42,7 +42,7 @@ module.exports = async function (options) {
             END) as subscribed
         `))
         .select(knex.raw(`
-            (SELECT GROUP_CONCAT(product_id) FROM members_products f WHERE f.member_id = members.id) as tiers
+            (SELECT GROUP_CONCAT(product_id,'~~',COALESCE(expiry_at,'')) FROM members_products f WHERE f.member_id = members.id) as tiers
```

Comment:
Since `members_products` column has both product id and its expiry now, this allows returning the product on member in format - `TIER_ID~~EXPIRY_AT`, where `EXPIRY_AT` is set as empty string if its null (for forever expiry).